lib/Shared.js

```diff
@@ -0,0 +1,197 @@
+var parseRange = require("range-parser");
+var pathIsAbsolute = require("path-is-absolute");
+var MemoryFileSystem = require("memory-fs");
+
+module.exports = function shared(context) {
+	var shared = {
```

Comment:
I've never been a fan of vars sharing the same name as the fn that contains them. Makes for some funky reading. :)